src/main/java/com/ontotext/trree/geosparql/conversion/CRSUtil.java

```diff
@@ -14,6 +17,10 @@ public class CRSUtil {
 	private static final String EPSG_PREFIX = "http://www.opengis.net/def/crs/EPSG/0/";
 	private static final String CRS84 = "http://www.opengis.net/def/crs/OGC/1.3/CRS84";
 
+	static {
+		Calculus.setCurrentNumberSystem(new DefaultNumberSystem());
```

Comment:
I assume this fixes the class loader issue by explicitly instantiating the class using the same class loader as the rest of the plugin. Please put a comment here on why it was needed.